### DISCUSSION TITLE
AInsights for Power BI - Enhance Your Azure AI Chat Deployments! 🔍📊

### DISCUSSION THREAD

**Discussion by xeekai** [microsoft/sample-app-aoai-chatGPT#1130] [General]
Hello everyone!

I’m Paul, also known as **CDR Guru**, from [Metropolis.com](https://metropolis.com), and I’m excited to introduce **AInsights for Power BI**, a tool that complements this Microsoft-managed repository for deploying Azure AI chat models. This tool allows you to **track conversations stored in Cosmos DB** and **visualize them in Power BI**, helping you gain deeper insights into the conversations generated by your Azure AI chatbots.

---

## 🔑 Key Features

- **Seamless Integration**: Enhance your Azure AI chat models by connecting them with **AI Insights for Power BI**.
- **Conversation Tracking with Cosmos DB**: Efficiently store and manage your chatbot conversations.
- **Powerful Visualizations in Power BI**: Transform raw data into actionable insights with interactive dashboards.
- **AI-Powered Analytics**: Leverage **Azure AI** to extract valuable insights from your conversation data.

---

## 🛠️ How It Works

With **AI Insights**, you can:

- **Monitor Customer Interactions**: Keep track of user engagement with your chatbot.
- **Analyze Sentiment and Key Topics**: Understand user sentiments and the main topics of discussion.
- **Visualize Real-Time Metrics**: Access up-to-date analytics through dynamic Power BI dashboards.

📽️ **[Watch our YouTube teaser video](https://www.youtube.com/watch?v=9D0i-J-KIa0)** to see the tool in action!

🤖 Try out the **[AInsights Assistant](https://chatgpt.com/g/g-bXFeEEjH4-ainsights-assistant)**—an AI tool that lets you interact with your data using natural language queries.

📥 **[Get AInsights for Power BI on AppSource](https://appsource.microsoft.com/en-us/product/power-bi/metropoliscorp1602190046889.chat-with-powerbi-analytics)** and start transforming how you manage your data today!

---

## 🤔 Let's Collaborate!

To make **AInsights for Power BI** even better, I'd love to hear your thoughts:

- **Share Your Experiences**: How are you integrating AI into your Power BI workflows?
- **Provide Feedback**: What features would enhance your experience with AI Insights?
- **Ask Questions**: If you have any queries or need help getting started, feel free to ask!

*Join the discussion below and let's collaborate to make this tool even better.*

---

## 🔥 Get Involved!

- **Star ⭐ the [GitHub Repository](https://github.com/microsoft/sample-app-aoai-chatGPT/)** and join the conversation.
- **Showcase Your Projects**: Share how you're using **AI Insights for Power BI**—I'd love to see your use cases!
- **Stay Updated**: I'll be sharing regular updates and hosting Q&A sessions, so stay tuned!

Let’s turn data into insights, together! 💡

---

Looking forward to your contributions,

**Paul (CDR Guru)**

[Paul's LinkedIn] (https://www.linkedin.com/in/paulmcdavis)
[Metropolis.com](https://metropolis.com)

---

**Comment by cdrguru** [comment]
Hi everyone! 👋

I wanted to share a video that provides more details about **AI Insights for Power BI** and its unique features:

📺 **[Watch the detailed demo here](https://www.youtube.com/watch?v=6D2FRLi6q8A)**

I’d love to hear your thoughts on how these features could benefit your projects or any ideas you have for improvements.

Let’s discuss and collaborate!

\- **cdrguru**
